### PR TITLE
Add account signers to Hubble 2.0

### PR DESCRIPTION
**Pull request by erika-sdf**

### CODE REVIEW

File: cmd/export_ledger_entry_changes.go [hunk 1]
```diff
@@ -99,9 +102,18 @@ be exported.`,
 							if err != nil {
 								entry, _, _, _ := utils.ExtractEntryFromChange(change)
 								cmdLogger.LogError(fmt.Errorf("error transforming account entry last updated at %d: %s", entry.LastModifiedLedgerSeq, err))
+							} else {
+								transformedOutputs["accounts"] = append(transformedOutputs["accounts"], acc)
+							}
```
Comment (sydneynotthecity) [inline cmd/export_ledger_entry_changes.go:107]: ❓  Should this be wrapped in an `else` statement? The rest of the `LedgerEntryType` append the `LedgerType` regardless if an error is returned.

**Review by sydneynotthecity — COMMENTED** [review]
1 question, and looks like a couple unit tests are failing. Once those pass, i'll approve

File: cmd/export_ledger_entry_changes.go (hunk 1 above)
Comment (erika-sdf) [reply cmd/export_ledger_entry_changes.go:107]: I think in other instances where there is an error during `Transform*`, the error check hits `continue`, and the next `ledgerEntry` is considered. 

In this case, both `signers` and `account` depend on the same `ledgerEntry`. I guess it depends on whether we still want to try to extract signers if `TransformAccount` returns an error. I think it's reasonable to just bail as well.

**Comment by erika-sdf** [comment]
This is ready for review again

File: internal/transform/account_signer.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	ledgerEntry, changeType, outputDeleted, err := utils.ExtractEntryFromChange(ledgerChange)
+	if err != nil {
+		return signers, err
+	}
+	outputLastModifiedLedger := uint32(ledgerEntry.LastModifiedLedgerSeq)
+	accountEntry, accountFound := ledgerEntry.Data.GetAccount()
+	if !accountFound {
+		return signers, fmt.Errorf("could not extract signer data from ledger entry of type: %+v", ledgerEntry.Data.Type)
+	}
+	sponsors := accountEntry.SponsorPerSigner()
```
Comment (sydneynotthecity) [inline internal/transform/account_signer.go:24]: Can there be multiple sponsors on an account? And if so, how are we parsing each sponsor individually?

File: cmd/export_ledger_entry_changes.go [hunk 3]
```diff
@@ -101,7 +104,18 @@ be exported.`,
 								cmdLogger.LogError(fmt.Errorf("error transforming account entry last updated at %d: %s", entry.LastModifiedLedgerSeq, err))
 								continue
 							}
-							transformedAccounts = append(transformedAccounts, acc)
+							transformedOutputs["accounts"] = append(transformedOutputs["accounts"], acc)
+							if change.AccountSignersChanged() {
```
Comment (sydneynotthecity) [inline cmd/export_ledger_entry_changes.go:108]: If the accounts signers change and that is the only change to the account, does the `accounts` data change at all? Or would we get a "changed" record that duplicated the last ledger modified sequence?

**Review by sydneynotthecity — APPROVED** [review]
Couple questions otherwise LGTM

File: internal/transform/account_signer.go (hunk 2 above)
Comment (erika-sdf) [reply internal/transform/account_signer.go:24]: I think there can be up to one sponsor per signer, and multiple signers per account.  The sponser per signer is extracted in the loop on L27.

File: cmd/export_ledger_entry_changes.go (hunk 3 above)
Comment (erika-sdf) [reply cmd/export_ledger_entry_changes.go:113]: I see. I added a conditional to only update `accounts` only if `AccountChangedExceptSigners`.

**Pull request merged**